--- whisper/triton_ops.py.orig	2025-05-22 14:08:23.116969492 -0500
+++ whisper/triton_ops.py	2025-05-22 14:09:14.882997993 -0500
@@ -60,7 +60,8 @@
         tl.store(y_ptr + offsets, MIDDLE_ROW_HERE, mask=mask)  # noqa: F821
 
     kernel = triton.JITFunction(kernel.fn)
-    kernel.src = kernel.src.replace(
+    src = kernel.src
+    src = src.replace(
         "    LOAD_ALL_ROWS_HERE",
         "\n".join(
             [
@@ -69,7 +70,7 @@
             ]
         ),
     )
-    kernel.src = kernel.src.replace(
+    src = src.replace(
         "    BUBBLESORT_HERE",
         "\n\n".join(
             [
@@ -90,7 +91,9 @@
             ]
         ),
     )
-    kernel.src = kernel.src.replace("MIDDLE_ROW_HERE", f"row{filter_width // 2}")
+    src = src.replace("MIDDLE_ROW_HERE", f"row{filter_width // 2}")
+    kernel._unsafe_update_src(src)
+    kernel.hash = None
 
     return kernel
 
